user_config.c

```diff
@@ -0,0 +1,73 @@
+#include "user_config.h"
+#include <stdio.h>
+
+static uint8_t set_debug_level = 0, change_function = 0, temp_num = 0;
```

Comment:
Please don't declare multiple variables on the same line. I know I've been guilty of doing it before, but it's much more readable with one per line